robot_name: notebook
ego_noise_sources:
  - name: cpu_fan
    type: fan
    frame: cfan